### PR TITLE
python-psutil - Update to 5.6.7 to fix CVE-2019-18874

### PR DESCRIPTION
**Pull request by nicogbg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update to 5.6.7 to fix CVE-2019-18874

###### Change Log  

spec and signature file

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2019-18874

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -6206,7 +6206,7 @@
         "other": {
           "name": "python-psutil",
           "version": "5.6.3",
```
Comment (mfrw) [inline cgmanifest.json:6208]: We missed this:
```suggestion
          "version": "5.6.7",
```

Comment (nicogbg) [reply cgmanifest.json]: good point !

Comment (nicogbg) [reply cgmanifest.json]: Just pushed the fix

**Review by mfrw — APPROVED** [review]
LGTM :)

**Pull request merged**